stan/math/fwd/fun/tcrossprod.hpp

```diff
@@ -1,11 +1,12 @@
 #ifndef STAN_MATH_FWD_FUN_TCROSSPROD_HPP
 #define STAN_MATH_FWD_FUN_TCROSSPROD_HPP
 
-#include <stan/math/prim/meta.hpp>
 #include <stan/math/prim/fun/Eigen.hpp>
+#include <stan/math/fwd/meta.hpp>
+#include <stan/math/fwd/fun/multiply.hpp>
 #include <stan/math/prim/fun/to_ref.hpp>
 #include <stan/math/prim/fun/transpose.hpp>
-#include <stan/math/fwd/fun/multiply.hpp>
+#include <stan/math/prim/fun/multiply.hpp>
```

Comment:
Was this meant to be a deletion?

Reply:
No sorry we are supposed to use `multiply` in this function.

multiply returns a scalar for `multiply(vector, row_vector)`. Should this function also return a scalar in the case of a vector?

Reply:
^yes. What about the second part of my Q. I think it should always return a matrix

Reply:
I think this should always return a Eigen::Matrix to match prim, rev, etc. so the latest looks correct?